### PR TITLE
Updates to CDC w/ DSE example, OSS C* & DSE versions

### LINKED ISSUE #132: CDC with DSE example docs needs to be updated
Post [version `2.0.0`](https://github.com/datastax/cdc-apache-cassandra/releases/tag/v2.0.0), certain files got removed/renamed/etc., and hence [this example documentation](https://docs.datastax.com/en/cdc-for-cassandra/docs/2.2.2/cdcExample.html) needs to be updated to match the current versions and assets.

### PR DESCRIPTION
**Pull request by msmygit**
Closed #132

### CODE REVIEW

**Comment by msmygit** [comment]
Reverting the CI image bumps per review comments

File: gradle.properties [hunk 1]
```diff
@@ -10,9 +10,9 @@ releasesRepoUrl=https://repo.datastax.com/artifactory/datastax-public-releases-l
 avroVersion=1.10.2
 lombokVersion=1.18.20
 ossDriverVersion=4.15.0
-cassandra3Version=3.11.10
```
Comment (aymkhalil) [inline gradle.properties:13]: Could you please revet those as well?

**Review by aymkhalil — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 1 above)
Comment (msmygit) [reply gradle.properties:13]: @aymkhalil are these causing any build failures?

Comment (aymkhalil) [reply gradle.properties]: No, but event the CI ones where not causing breaking issues. The idea was to keep this PR scope limited to docs so we can discuss dependency version independently and unblock this PR faster.

**Review by aymkhalil — APPROVED** [review]

**Comment by aymkhalil** [comment]
@mendonk the doc publish tasks failed with https://github.com/datastax/cdc-apache-cassandra/actions/runs/4632552208
```
Task 'antoraConfig' not found in project ':docs'.
```

I've seen this error with previous doc publishing PRs, any idea?

**Pull request merged**